apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data.IntegrationTests/TestDatabaseFixture.cs

```diff
@@ -136,31 +140,36 @@ public void Dispose()
                                             Key                STRING(MAX) NOT NULL,
                                             StringValue        STRING(MAX),
                                           ) PRIMARY KEY (Key)";
+            string createDataTableStatement = $@"CREATE TABLE {DataAdapterTestTable} (
+                                            Key                STRING(MAX) NOT NULL,
+                                            StringValue        STRING(MAX),
+                                          ) PRIMARY KEY (Key)";
+
             var index1 = "CREATE INDEX TestTableByValue ON TestTable(StringValue)";
```

Comment:
No I'd prefer to keep them distinct for now.  Its actually going to create more lines of code and possibly be harder to read.